app/views/fields/user_submitter_field/_show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+Yanked the entire first section of this code from
+https://github.com/thoughtbot/administrate/blob/master/app/views/fields/has_many/_show.html.erb
+%>
+<%#
+# HasMany Show Partial
+This partial renders a has_many relationship,
+to be displayed on a resource's show page.
+By default, the relationship is rendered
+as a table of the first few associated resources.
+The columns of the table are taken
+from the associated resource class's dashboard.
+## Local variables:
+- `field`:
+  An instance of [Administrate::Field::HasMany][1].
+  Contains methods to help display a table of associated resources.
+[1]: http://www.rubydoc.info/gems/administrate/Administrate/Field/HasMany
+%>
+<% if field.resources.any? %>
+  <% order = field.order_from_params(params.fetch(field.name, {})) %>
+  <% page_number = params.fetch(field.name, {}).fetch(:page, nil) %>
+  <%= render(
+    "collection",
+    collection_presenter: field.associated_collection(order),
+    collection_field_name: field.name,
+    page: page,
+    resources: field.resources(page_number, order),
+    table_title: field.name,
+  ) %>
+  <% if field.more_than_limit? %>
+    <%= paginate field.resources(page_number), param_name: "#{field.name}[page]" %>
+  <% end %>
+<% else %>
+  <%= t("administrate.fields.has_many.none", default: "–") %>
+<% end %>
+
+<br />
```

Comment:
I think we should remove this `br` tag and wrap the 'new submitter' button in a `div` with a top margin of 1rem:
![Screen Shot 2021-03-11 at 11 00 12 AM](https://user-images.githubusercontent.com/16103405/110818272-1d4aac80-825b-11eb-81f4-2fee47f3274a.png)

This would more closely match the spacing when submitters are present:
![Screen Shot 2021-03-11 at 11 00 46 AM](https://user-images.githubusercontent.com/16103405/110818471-4703d380-825b-11eb-943d-3836351e3af5.png)

Reply:
But I'm lazy and <br /> fits that well. Also, you are absolutely correct and I'll adjust. Thanks!